README.md

```diff
@@ -1,8 +1,22 @@
 # wp-cldr
 
-WordPress plugin to access localized territory and language names, currency names/symbols, and other localization info. Source is the [JSON distribution] (https://github.com/unicode-cldr/cldr-json) of the [Unicode Common Locale Data Repository (CLDR)] (http://cldr.unicode.org/).
+This plugin provides WordPress developers with easy access to localized country/region names, language names, currency names/symbols/usage, and other localization info from the [Unicode Common Locale Data Repository (CLDR)] (http://cldr.unicode.org/).
 
-This repository includes [locale display names] (https://github.com/unicode-cldr/cldr-localenames-modern) and [number formatting] (https://github.com/unicode-cldr/cldr-numbers-modern) for the locales currently [used by WordPress.com](https://github.com/Automattic/wp-cldr/blob/master/prune-cldr-files.php#L18) and [used by WordPress.org](https://github.com/Automattic/wp-cldr/blob/master/prune-cldr-files.php#L29), as well as the [core set of supplemental data] (https://github.com/unicode-cldr/cldr-core).
+With the plugin active, WordPress developers will have access across almost 100 WordPress locales to localized data items including:
+- localized names for territories including ISO 3166 country codes and UN M.49 region codes
+- localized currency names and symbols for ISO 4317 currency codes
+- information on currency usage in different countries
+- localized language names for ISO 639 language codes
+- localized calendar information including the first day of the week in different countries
+- telephone codes for different countries
+
+The plugin includes support for high volume application. It includes two layers of caching (in-memory arrays and the WordPress object cache). It is currently used on WordPress.com.
+
+CLDR is a library of localization data managed by Unicode. It emphasizes [common, everyday usage] (http://cldr.unicode.org/translation/country-names) and is available in over 700 language-region locales. It is [updated every six months] (http://cldr.unicode.org/index/downloads) by its [users who include Google, Apple, Yahoo, Microsoft, Ebay, Apache, Debian Linux, Free BSD, Mozilla, Wikipedia] (http://cldr.unicode.org/#TOC-Who-uses-CLDR-). CLDR data is licensed under [Unicode's data files and software license] (http://unicode.org/copyright.html#Exhibit1) which is on [the list of approved GPLv2 compatible licenses] (https://www.gnu.org/philosophy/license-list.html#Unicode).
```

Comment:
I'd limit the "Who uses CLDR" link to the word `users` & leave the remainder of the sentence unlinked (`who include G...`)